tasks/post_copy_request_to_queue/bin/build.sh

```diff
@@ -108,6 +108,13 @@ let return_code=$?
 
 check_rc $return_code "ERROR: Failed to copy lambda files to build directory."
 
+## Copy the schema files to build
+echo "INFO: Copying schema files ..."
+cp -r schemas/ build/
+let return_code=$?
+
+check_rc $return_code "ERROR: Failed to copy schema files to build directory."
+
```

Comment:
I'm guessing this was using a non-standard method of pulling in schema files, and I messed it up when I removed the redundant copies.